hal_ptu:
  ros__parameters:
    hz: 10.0

    # Ranges for Pan/Tilt. 
    limits.min_tilt: -0.3
    limits.max_tilt:  0.3
    limits.tilt_speed: 0.1
    limits.min_pan: -0.7
    limits.max_pan: 0.7
    limits.pan_speed: 0.1

    publishers:
      state: /ptu/state

    services:
      set_pan: /ptu/set_pan
      set_tilt: /ptu/set_tilt
      set_pantilt: /ptu/set_pan_tilt
      set_pantilt_speed: /ptu/set_pan_tilt_speed
      reset: /ptu/reset
      get_limits: /ptu/get_limits

    actions: 
      set_pan: /ptu/set_pan
      set_tilt: /ptu/set_tilt
      set_pantilt: /ptu/set_pan_tilt

    min_thresold_command_input_pan: 0.01
    min_thresold_command_input_tilt: 0.01